.evergreen/run-mongodb-aws-test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Don't trace since the URI contains a password that shouldn't show up in the logs
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Supported/used environment variables:
+#       MONGODB_URI             Set the URI, including an optional username/password to use to connect to the server via MONGODB-AWS
+#                               authentication mechanism
+#       JDK                     Set the version of java to be used.  Java versions can be set from the java toolchain /opt/java
+#                               "jdk5", "jdk6", "jdk7", "jdk8", "jdk9", "jdk11"
+
+JDK=${JDK:-jdk11}
+MONGODB_URI=${MONGODB_URI:-"mongodb://localhost"}
+
+############################################
+#            Main Program                  #
+############################################
+
+echo "Running MONGODB-AWS authentication tests"
+
+export JAVA_HOME="/opt/java/${JDK}"
+
+MONGODB_URI="${MONGODB_URI}/aws?authMechanism=MONGODB-AWS&authSource=\$external"
```

Comment:
I don't think the `/aws` is necessary.  Can just be:

    MONGODB_URI="${MONGODB_URI}/?authMechanism=MONGODB-AWS&authSource=\$external"

Also not sure if the authSource is required. If it defaults to $external anyway, you can just remove it.

Reply:
Removed the authSource since that defaults to external. For the database I was just keeping in-line with your other .groovy test and pulling the database from the URI as such: `executeCommand(getConnectionString().getDatabase(), new BsonDocument('count', new BsonString('test')), connection)`. Would you rather I just hard-code the database name throughout the test?

Reply:
No, it's fine this way.  Didn't realize that was why you were doing this.